docs/resources.rst

```diff
@@ -21,6 +21,8 @@ Solidity Integrations
 
 * Generic:
 
+    * `Superblocks Lab <https://lab.superblocks.com/>`_
+        Browser-based IDE. Built in browser blochain VM, Metamask integration (one click deployments to Testnet/Mainnet), transaction logger and WebApp live coding using with Web3.
```

Comment:
The syntax is a little off towards the end. Also note that this list is sorted alphabetically - could you please move it to the end?

Reply:
Hi @chriseth yes of course! Let me do the changes :)

Reply:
"blochain" seems to be a typo